python/samples/getting_started_with_agents/chat_completion/step12_chat_completion_agent_code_interpreter.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+import datetime
+import os
+
+from azure.core.credentials import AccessToken
+from azure.core.exceptions import ClientAuthenticationError
+from azure.identity import DefaultAzureCredential
+
+from semantic_kernel.agents import ChatCompletionAgent
+from semantic_kernel.connectors.ai.open_ai import AzureChatCompletion
+from semantic_kernel.contents import ChatMessageContent, FunctionCallContent, FunctionResultContent
+from semantic_kernel.core_plugins import SessionsPythonTool
+
+"""
+The following sample demonstrates how to create a chat completion agent with
+code interpreter capabilities using the Azure Container Apps session pool service.
+"""
+
+auth_token: AccessToken | None = None
+
+ACA_TOKEN_ENDPOINT: str = "https://acasessions.io/.default"
+
+
+async def auth_callback() -> str:
```

Comment:
There's a [default auth callback](https://github.com/microsoft/semantic-kernel/blob/d7a76993876b0bd6f325d92d61b00109c43744a1/python/semantic_kernel/core_plugins/sessions_python_tool/sessions_python_plugin.py#L68C9-L69C70) included with the plugin - can we rely on that? It simplifies the sample quite a bit.